_tutorials/new-component/create-a-new-component-part-5.md

```diff
@@ -0,0 +1,300 @@
+// `data-o-component="o-example"` attribute.
+import oExample from 'o-example';
+oExample.init();</code></pre>
+
+Or a user could initialise a specific element, or its child elements, by calling the `init` method with an argument
+
+<pre><code class="o-syntax-highlight--js">// Initialise the `.my-selector` element or any of its children
+// which have the `data-o-component="o-example"` attribute.
+import oExample from 'o-example';
+const myElement = document.querySelector('.my-selector');
+oExample.init(myElement);</code></pre>
+
+<pre><code class="o-syntax-highlight--js">// Initialise the `.my-selector` element or any of its children
+// which have the `data-o-component="o-example"` attribute.
+import oExample from 'o-example';
+oExample.init('.my-selector');</code></pre>
+
+For more details see the [JavaScript initialisation](/spec/v1/javascript/#initialisation) section of the Origami specification.
+
+## User Configuration
+
+The second `init` argument is `opts`, an `Object` of options for the user to configure the component. So users of [o-autoinit](https://registry.origami.ft.com/components/o-autoinit@2.0.4/readme) or the [Origami Build Service](https://www.ft.com/__origami/service/build/v2/) can also configure components, data attributes may alternatively be used to set component configuration.
+
+In `/src/js/example` setting component configuration is handled in the constructor. The `this.options` property is assigned to the given `opts` object, which is merged with any data attributes that have a namespace `data-o-example-[option]`.
+
+For instance the `o-table` component has a sort feature which may be disabled by either passing `{sortable: false}` to the [`o-table` `init` method](https://registry.origami.ft.com/components/o-table@8.0.11/jsdoc?brand=master) or by adding the [`data-o-table-sortable="false"`](https://registry.origami.ft.com/components/o-table@8.0.11/readme?brand=master#disable-sort) attribute to the `o-table` element.
+
+We'll add configuration options later to demonstrate. For full details see the [JavaScript configuration](/spec/v1/javascript/#configuration) section of the Origami specification.
+
+## Interactivity
+
+Lets start work on making our example component interactive.
+
+We'll start by adding a `count` property, and listen for clicks on `o-example` buttons using the [`handleEvent`](https://medium.com/@WebReflection/dom-handleevent-a-cross-platform-standard-since-year-2000-5bf17287fd38) method to increment the count property.
+
+Origami components use browser apis directly for [Document Object Model](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) manipulation. For instance [`Document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) to get an element and [`HTMLElement.innerText`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText) to set an elements text content.
```

Comment:
For some reason, I have a feeling more people know the acronym DOM than "Document Object Model". Shall we change it to an abbreviation tag?
```suggestion
Origami components use browser apis directly for [<abbr title="Document Object Model">DOM</abbr>](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) manipulation. For instance [`Document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) to get an element and [`HTMLElement.innerText`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText) to set an elements text content.
```